submissions/blessingOlusuji/html-forms/product.html

```diff
@@ -0,0 +1,71 @@
+<!Doctype html>
```

Comment:
You are not supposed to create a new product page again. You already have one before. it is that one that you will link on the `<nav>`

Reply:
I didnt create a new one, I just pasted the existing one in the html-forms for easy access, since we were asked to save it in html-forms too.
I created product link on the product page itself because we were asked to create product and contact form links on both pages

Reply:
Modification done